dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -174,14 +174,13 @@ int SHELL_ConfirmMsgBox(HWND hWnd, LPWSTR lpszText, LPWSTR lpszCaption, HICON hI
 
 static const shvheader DesktopSFHeader[] = {
     {IDS_SHV_COLUMN_NAME, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 15},
-    {IDS_SHV_COLUMN_COMMENTS, SHCOLSTATE_TYPE_STR, LVCFMT_LEFT, 10},
     {IDS_SHV_COLUMN_TYPE, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 10},
     {IDS_SHV_COLUMN_SIZE, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_RIGHT, 10},
-    {IDS_SHV_COLUMN_MODIFIED, SHCOLSTATE_TYPE_DATE | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 12},
+    {IDS_SHV_COLUMN_MODIFIED, SHCOLSTATE_TYPE_DATE | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 15},
     {IDS_SHV_COLUMN_ATTRIBUTES, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 10}
 };
 
-#define DESKTOPSHELLVIEWCOLUMNS 6
+#define DESKTOPSHELLVIEWCOLUMNS 5
```

Comment:
Maybe
```suggestion
#define DESKTOPSHELLVIEWCOLUMNS RTL_NUMBER_OF(DesktopSFHeader)
```
Maybe even get rid of this define?

Reply:
I added the RTL.. for now. Thanks.

Reply:
@SergeGautherie, this is now handled in CFSFolder.cpp with this statement at about Line 536:
#define GENERICSHELLVIEWCOLUMNS _countof(GenericSFHeader)